pipelines/release-to-github/README.md

```diff
@@ -17,6 +17,10 @@ Tekton release pipeline to release binaries extracted from the image built with
 | taskGitUrl | The url to the git repo where the release-service-catalog tasks to be used are stored | Yes | https://github.com/redhat-appstudio/release-service-catalog.git |
 | taskGitRevision | The revision in the taskGitUrl repo to be used | No | - |
 
+### Changes in 1.2.0
+- modified the pipeline to dynamically source the `snapshot_spec.json`
```

Comment:
Capital M to match the other entries please

Reply:
Fixed, thanks!